src/content/basics/kubernetes-resources/index.md

```diff
@@ -5,15 +5,16 @@ date = "2020-02-03"
 weight = 140
 type = "page"
 categories = ["basics"]
+last-review-date "2020-03-31"
```

Comment:
```suggestion
last_review_date = "2020-03-31"
```

There was an equals sign missing. In addition we have an inconsistency in the repo regarding `last_review_date` vs. `last-review-date`. I would prefer `last_review_date`.

Reply:
I prefer keeping it with what appears the most of the docs which is `last-review-date`. Or else it'll just get messy and inconsistent.

Reply:
Now I'm stuck on how to update just part of the suggestion.